### PR TITLE
Dlg RowNamesOrNumbers 

### PR DESCRIPTION
**Pull request by stevenndungu**
Disenabling the OK button whenever the ucrRreceiver is empty and vice versa when non-empty.
This is when rhe radio button Set Row Names from Columns radio button is clicked.

### CODE REVIEW

File: instat/dlgRownamesOrNumbers.vb [hunk 1]
```diff
@@ -88,6 +90,13 @@ Public Class dlgRowNamesOrNumbers
             grpSortOptions.Visible = False
             lblColumnName.Visible = False
             ucrNewColumnName.Visible = False
+            If rdoCopyfromColumn.Checked Then
+                If Not ucrReceiverSingleRownamesOrNumbers.IsEmpty Then
+                    ucrBase.OKEnabled(True)
+                Else
+                    ucrBase.OKEnabled(False)
+                End If
+            End If
```
Comment (dannyparsons) [inline instat/dlgRownamesOrNumbers.vb]: This should be in TestOKEnabled, why did you want to put it here?

Comment (stevenndungu) [inline instat/dlgRownamesOrNumbers.vb]: I did that initially but did not work. That's when I resorted to this option

Comment (stevenndungu) [inline instat/dlgRownamesOrNumbers.vb]: Reason:
 If crSelectorByDataFrameAddRemoveforRownamesOrNumbers.ucrAvailableDataFrames.cboAvailableDataFrames.Text <> "" Then
            ucrBase.OKEnabled(True)
        Else
            ucrBase.OKEnabled(False)
End If

Which makes sure test OK enabled whenever there are selector is not empty. Thus putting it there its overwritten.

Comment (dannyparsons) [inline instat/dlgRownamesOrNumbers.vb]: Just getting the dialog to work is not good enough, the code needs to be done sensibly. Setting OKEnabled outside of the Test method is not sensible. What you can do is edit the TestOK method to how you need it and call it from other places. Can you look at this with @FrancoisJRenaud

**Comment by dannyparsons** [comment]
Looks good, well done.

**Pull request merged**